recipes/imagecodecs-lite/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2019.4.20" %}
+
+package:
+  name: imagecodecs-lite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/imagecodecs-lite/imagecodecs-lite-{{ version }}.tar.gz
+  sha256: 60e135d25325e5c2ec122eb578026aaac4fd2559fb99afa084ed85f4ce43e8b2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy 1.14.*
```

Comment:
This doesn’t need a version specifier, the default is 1.14.

Reply:
Yeah. I thought so too but see the Windows failure in the first commit.

Reply:
I'll remove it in the feedstock. It is probably some bad outdated AppVeyor config. Thanks @chrisburr!